README.md

```diff
@@ -1,54 +1,69 @@
 </p>
 <br/>
 
 ---
 
-No more plaintext credentials in your home directory. Automatically authenticate every CLI you use with [1Password CLI](https://developer.1password.com/docs/cli/) + [Shell Plugins](https://developer.1password.com/docs/cli/shell-plugins/). Approve temporary credential usage in your terminal with biometrics.
-
-## 🪄 Usage
+Eliminate plaintext credentials in your home directory. Automatically authenticate every CLI you use with [1Password CLI](https://developer.1password.com/docs/cli/) + [Shell Plugins](https://developer.1password.com/docs/cli/shell-plugins/). Approve temporary credential usage in your terminal with biometrics.
 
- <picture>
-   <source srcset="https://developer.1password.com/videos/aws.webp" type="image/webp" />
-   <img src="https://user-images.githubusercontent.com/7430639/205337855-41604aca-0ddb-4eab-a5f0-fb9107e09d8d.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
+## 🪄 Quick demo
+<p align="center">
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/45081667/227191964-9629476d-a49e-475d-b8cb-2115c302025d.gif">
+   <img src="https://user-images.githubusercontent.com/45081667/227197994-6fdb2cad-c240-4cb7-ba4c-77f2483606ab.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
 </picture>
+</p>
 
-## 🚀 Get started
 
-* Get started with [1Password Shell Plugins](https://developer.1password.com/docs/cli/shell-plugins)
-* Get started with [creating your own plugins](https://developer.1password.com/docs/cli/shell-plugins/contribute) <sup><b>[BETA]</b></sup>
+## 🚀 Get started with the available plugins
+<p align="center">
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/45081667/226968760-70d3f6b0-a3eb-4c75-a674-6fd136d7149a.png">
+  <img alt="Picture changing depending on light mode." src="https://user-images.githubusercontent.com/45081667/226969008-0a3f7537-7942-442f-9170-18b008a6574c.png">
+</picture>
+</p>
 
-## 👫 Contributing
+<a href="https://developer.1password.com/docs/cli/shell-plugins">
+    <p align="center">See all...</p>
+  </a>
+<br/>
 
-Is your favorite CLI not listed yet? Learn how to [build a shell plugin](https://developer.1password.com/docs/cli/shell-plugins/contribute) yourself and [open a PR](https://github.com/1Password/shell-plugins/pulls)!
+## 🔩 Don’t see yours? Contribute! <sup><b><a href="#-beta-notice">[BETA]</a></b></sup>
+Is your favorite CLI not listed yet? Learn [how to build a new plugin](https://developer.1password.com/docs/cli/shell-plugins/contribute) yourself and then open a PR on this repository to get it added to 1Password!
```

Comment:
"added to 1Password" sounds a bit vague to me. A way to clarify this would be something like:

```suggestion
Is your favorite CLI not listed yet? Learn [how to build a new plugin](https://developer.1password.com/docs/cli/shell-plugins/contribute) yourself and then open a PR on this repository to get it included in the 1Password CLI!
```

Reply:
Done!